### DISCUSSION TITLE
Saving signature does not work with iOS/Safari

### DISCUSSION THREAD

**Discussion by lennartwalter** [szimek/signature_pad#619] [General]
Everything works fine but saving with Safari. A new Tab opens, that`s all, even with the demo here ...

**Comment by szimek** [comment]
It's a bug in how the demo handles Safari, as it used to work differently in older Safari versions. Remove this condition https://github.com/szimek/signature_pad/blob/master/docs/js/app.js#L43 (leave only the part inside `else` clause) and see if it works then.

**Reply by lennartwalter** [reply]
Thanks a lot => it works ! Tried a modification at this point before, but didn't try to erase it completely 🙃
Looking forward to the world gettin' fed up with Apple 🤭
